tinkerbell/stack/templates/nginx.yaml

```diff
@@ -27,11 +27,14 @@ spec:
       containers:
       - name: {{ .Values.stack.name }}
         image: {{ .Values.stack.image }}
-        command: ["/bin/bash", "-c"]
+        command: ["/bin/bash", "-xeuc"]
         args:
-        - export POD_NAMESERVER=$(awk 'NR==2 {print $2}' /etc/resolv.conf);
-          envsubst '$POD_NAMESERVER' < /tmp/nginx.conf.template > /etc/nginx/nginx.conf;
-          nginx -g 'daemon off;'
+        - |
+          POD_NAMESERVER=$(awk 'NR==2 {print $2}' /etc/resolv.conf) \
+            envsubst '$POD_NAMESERVER' \
+            </tmp/nginx.conf.template \
+            >/etc/nginx/nginx.conf
+          exec nginx -g 'daemon off;'
```

Comment:
I think this is simpler with separate commands, particularly because of the stdin and stdout in the middle.

Reply:
Not sure what you mean. It's using a regular env=value command syntax and prevents environment variable leaking into nginx.

Reply:
Sorry, I overlooked the `- |` and its impact to new lines.